libs/sdk-ui-charts/styles/scss/tooltip.scss

```diff
@@ -1,28 +1,17 @@
 // (C) 2007-2020 GoodData Corporation
-@import "~@gooddata/goodstrap/lib/Bubble/styles/mixins";
-@import "~@gooddata/goodstrap/lib/core/styles/variables";
+@import "~@gooddata/sdk-ui-kit/styles/scss/variables";
+@import "~@gooddata/sdk-ui-kit/styles/scss/mixins";
 @import "variables";
```

Comment:
I believe that sdk-ui-kit variables are being imported from `@import "variables";` line.